__tests__/app.test.js

```diff
@@ -113,4 +127,31 @@ describe("NC News Server", () => {
         });
     });
   });
+  describe("GET /apiarticles/:article_id/comments", () => {
+    test('GET 200 : return all comments for a specific article_id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.comments.length).toBe(11)
+            body.comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe('number')
+                expect(typeof comment.votes).toBe('number')
+                expect(typeof comment.created_at).toBe('string')
+                expect(typeof comment.author).toBe('string')
+                expect(typeof comment.body).toBe('string')
+                expect(typeof comment.article_id).toBe('number')
+            })
+        })
+    })
+    test('GET 200 : return all comments for a specific article_id in descending order by date comment was posted', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.comments.length).toBe(11)
+            expect(body.comments).toBeSortedBy('created_at', { descending: true })
+        })
+    })
```

Comment:
What about an article that has no comments? What would that status code be? (Article 2 has no comments)